cmake/Finddpdk.cmake

```diff
@@ -153,7 +153,7 @@ if (dpdk_FOUND AND NOT (TARGET dpdk))
   if(_warning_supported_volatile)
     # include/generic/rte_spinlock.h increments volatiled-qualified type with
     # "++". but this is deprecated by GCC, so silence it.
-    set(compile_options
+    set(dpdk_compile_options
```

Comment:
we should not tolerate this name clashing. so i created #2548 to address this.

Reply:
Dropped this commit after https://github.com/scylladb/seastar/pull/2548 landed.